test/e2e-tests.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+failed=0
+source $(dirname $0)/../vendor/knative.dev/hack/e2e-tests.sh
+
+HELM_BIN="/tmp/helm"
+KEDA_NS="keda"
+
+initialize --num-nodes=4 --cluster-version=1.28 "$@"
+
+git clone https://github.com/knative/serving.git "serving"
```

Comment:
I am not bringing in tests as deps, just doing what we do on Kind. Bringing in deps (eg. hack/tools) would add a lot of not required stuff to this repo.